tools/validators/instance_validator/README.md

```diff
@@ -5,6 +5,8 @@ The Instance Validator allows validation of YAML instance files to make sure the
 ## Usage
 The Instance Validator is written in Python3 and takes as an argument the path pointing to the instance files.
 
-The validator can be run as following: `python3 main.py path/to/test_file.yaml`
+To use to ontology defined in this repository, run the validator as following: `python3 main.py --input path/to/YOUR_BUILDING_CONFIG.yaml`
+
+If you want to use a modified ontology, run the following: `python3 main.py --input path/to/test_file.yaml --modified-ontology-types path/to/modified/ontology/types/folder`
```

Comment:
rename `main.py` to `instance_validator.py`

update the changes in the:

1. Setup.py file
2. rename the main to this new name

 setup.py file

Reply:
Done